__init__.py

```diff
@@ -172,10 +172,12 @@ def pkg_file(*name):
                 DYLDFLAGS=
                 '''), 'ascii'))
 
-        # Manually merge Spicy analyzer-specific changes to `scripts/__load__.zeek`.
-        with open(pkg_file('scripts', '__load__.zeek'), 'ab') as f:
-            f.write(b'@load-sigs ./dpd.sig\n')
-
+        # Manually remove files we don't need.
```

Comment:
I'm torn about these tests, btw. From examples of packages based on the template that I've seen so far, it's not clear to me that they're that useful (i.e., people leave them in place unchanged even if running `http.pcap` really doesn't make much sense). I could see making those tests template features, too... then they wouldn't need removal here.

Reply:
Yeah, true, although the new traces for Spicy analyzers that I am adding will have the same issue. Generally I think it's better to have some trace so that there's an end-to-end example.